app/views/spree/orders/_pay_easy.html.erb

```diff
@@ -0,0 +1,8 @@
+<div>
+  Thank you for your order. Instructions have been sent to your e-mail address.<br>
+  To complete a payment at <strong><%= link_to "purchase page", source.pay_url %></strong>.<br>
```

Comment:
The English should be something like: `"To complete your payment, visit the <strong><%= link_to "payeasy online banking page", source.pay_url %></strong>."`